### ISSUE TITLE
JwtHeaders return hexa structure in headers

### ISSUE THREAD

**Issue by anmorel** [team-carepay/traefik-jwt-plugin#3]
Hello,

I want return JWT data in headers with JwtHeaders option and the value of header is hexa structure instead of real value.

My config is : 
```
JwtHeaders:
         Subject: sub
         User: preferred_username
```

Header returned is : 
```
GET / HTTP/1.1
Host: whoami.xxxxxxxx.net
User-Agent: PostmanRuntime/7.26.10
Accept: */*
Accept-Encoding: gzip, deflate, br
Authorization: Bearer xxxxxxxxxxxxxxxxxxxxxxxxx
Subject: {0xc000495400 {0x2cbc100 0xc0008c1a90 152}}
User: {0xc000495400 {0x2cbc100 0xc0008c1c20 152}}
X-Forwarded-For: xx.xx.xx.xx
X-Forwarded-Host: whoami.xxxxxxxx.net
X-Forwarded-Port: 443
X-Forwarded-Proto: https
X-Forwarded-Server: traefik-5b747b5845-mlw7r
X-Real-Ip: xx.xx.xx.xx
```

My JWT token is here [JWT.IO SHARE](https://jwt.io/#debugger-io?token=eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICI5b29vQmZNZldjYUowVEl6SmhPeUs3cXJqNklrbnF2NTZRNlhTRTM5SlFvIn0.eyJleHAiOjE2MTkyMTQ3MjIsImlhdCI6MTYxOTIxNDQyMiwianRpIjoiMDQxNDE4MTUtMjlmMy00OGVlLWI0ZGQtYTA0N2Q1NWU1MjcxIiwiaXNzIjoiaHR0cHM6Ly9rZXljbG9hay50ZXN0LnNjdy5mcmVlcGhwNS5uZXQvYXV0aC9yZWFsbXMvdGVzdCIsImF1ZCI6ImFjY291bnQiLCJzdWIiOiJjMDNhM2Q4YS1lMGI1LTQ3Y2EtOWIwZi1iMmY5ZTY5Y2YzNDgiLCJ0eXAiOiJCZWFyZXIiLCJhenAiOiJ0ZXN0LWNsaWVudCIsInNlc3Npb25fc3RhdGUiOiJjMmU1MmFhYS0yOTVkLTRhOWItOGNmMS1iYmIyYzliZmVmMmEiLCJhY3IiOiIxIiwiYWxsb3dlZC1vcmlnaW5zIjpbImh0dHBzOi8vd2hvYW1pLnRlc3Quc2N3LmZyZWVwaHA1Lm5ldCJdLCJyZWFsbV9hY2Nlc3MiOnsicm9sZXMiOlsib2ZmbGluZV9hY2Nlc3MiLCJ1bWFfYXV0aG9yaXphdGlvbiJdfSwicmVzb3VyY2VfYWNjZXNzIjp7ImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyIsInZpZXctcHJvZmlsZSJdfX0sInNjb3BlIjoiZW1haWwgcHJvZmlsZSIsImVtYWlsX3ZlcmlmaWVkIjpmYWxzZSwicHJlZmVycmVkX3VzZXJuYW1lIjoidXNlciJ9.aRNE0vl8YsVSUqefM8vXpeuX_uQT5iJbO_RKvTD4fePOfjL8n-uxhUDh4RpBJ_eDGBi2e2PQAQ2tP0epSfSG6uY48ypjRULpRla5Qdnu414VaWvHT4OFyHMYQ3PfiVlNf7-sHkhDEkEy_0LZD1y2Vv6Wndb7_DmxUXh9GAIS5KPrOX9qAW88MBZNvv2Hn-qgAEk9VxLlCt1o6vGkX2xnfymW78TaY47X4UTKnMhvn8crp7VFnn2ZGcF-dHYPUsI-jjfuoLz5GbbUuWG2kXyqRAtdB2qFGWH6Lsv9PxAVnL0YTiIimMGnnT9GGCAgoqjzPc2jJ6vzACL7CW8J6IXnsg&publicKey=-----BEGIN%20PUBLIC%20KEY-----%0AMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAnN070angKehIpiHaFtyq%0Aio9mQlYIk%2FYHeXGYwoJr9fufvKp7aermZDBwLa0rK%2FdzFUrLJBGSDWh9N0ffvvjw%0Ao6SPVuzKMYfCaHwXWW91NlxZqyf3PNLsDI02CLhkA0jrlSwZGVTjQrnBfAHaLT0e%0AELlDhHV1G0DxCUZ0xAMi3aem%2B4wK%2BtKOSsJbyfghy7eEd7m8eJ%2B%2BnmOkwaookQ23%0Am5Qtb4ThPzjiqf0yZ0ju9zIq4oSa4gQuFjcv0KPDOquT7tFGQ0t3E7UMao0iy%2FMW%0ACNHw7O1yufw227PeleDHIDm9TsRqjyoDjrDlzexAp1KShrJXzvUlnwmYRZGlkQg7%0AxwIDAQAB%0A-----END%20PUBLIC%20KEY-----%0A) 

It's a bug ? or no ?

Thx

**Comment by blagerweij** [comment]
Hey @anmorel, thanks for you bug-report. Your test-case should be covered by a unit-test, but somehow the plugin acts differently when used in Traefik itself.

Anyway, I have added a test-case, and made a small code change. Your issue should be resolved in v0.0.6, please give that a try

**Comment by blagerweij** [comment]
Closing the issue...
